lib/medium-admin.php

```diff
@@ -136,6 +136,17 @@ public static function add_meta_boxes_post($post) {
   public static function save_post($post_id, $post) {
     if (defined('DOING_AUTOSAVE') && DOING_AUTOSAVE) return;
 
+    $allowed_post_types = (array) apply_filters('medium_allowed_post_types', array('post'));
+
+    if (!in_array($post->post_type, $allowed_post_types)) {
+      return;
+    }
+
+    // Use the ID of the parent post if this is a post revision
+    if ($parent_id = wp_is_post_revision($post)) {
```

Comment:
@majelbstoat However I think we may not need to do this check at all because it will be solved by the `$allowed_post_types` array, since `revision` is actually a post type. Pretty fun, eh?! :smiley_cat: